cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/cluster/KafkaClusterOperations.java

```diff
@@ -102,7 +122,35 @@ public void create(String namespace, String name, Handler<AsyncResult<Void>> han
 
             result.add(statefulSetOperations.create(kafka.generateStatefulSet(isOpenShift)));
 
-            return CompositeFuture.join(result);
+            // TODO :
```

Comment:
This should be async, so I don't think it matters if we wait even when TC is not deployed. In fact I though this might be good anyway, because right now we print into log that Kafka cluster is created even when it is still booting.